### ISSUE TITLE
The provided locale `sr-CS` is invalid.

### ISSUE THREAD

**Issue by Zoran23** [jquery-validation/jquery-validation#2282]
### Subject of the issue
I translated prestashop 1.7.5 about 70%. Then I get to know that this translation was mix of Serbian Latin and Serbian Cyrillic letters. 
Another problem with Serbian language was with currency. Adding currency in prestashop 1.7.5 with Serbian language did not work because of some problem with locale. 1.7.5. Serbian was massy and gave up with translation and was just waiting for new version 
Now when prestashop 17.6 beta is up, I have tested it and translated online everything 100%. (Only Latin)

Now in 1.7.6 currency is working fine and we have complete Serbian latin translation, but new big problem came. It is not possible to translate Serbian language in BO. Error that is coming is following: The provided locale `sr-CS` is invalid.
Yes I have translated it 100% online, but still maybe someone want to change something through back office. 
If someone can check up why this error, “The provided locale `sr-CS` is invalid”, coming up when try to translate Serbian language on BO. 
Thanks in advance. 
Zoran

### Your environment
* version of `jquery-validate`:
* which browser and its version:

**Comment by zavela** [comment]
Have you maybe found a solution to this problem or does it have to manually

Thx

**Comment by Zoran23** [comment]
Yes, I found solution for the problem
Locale SR-CS is wrong, it should be called SR-RS (Combination of country code and language code according to some lines in prestashop cod)
I had to rename translation folder SR-CS in SR-RS and in all translations file name I changed SR-CS in SR-RS. Then even in all translation files content I changed Sr-CS in Sr-RS.
Then In this folder : \localization\CLDR\core\common\main I  added sr_RS.xml file and another file sr.xml I modified, changing SR-CS in SR-RS
In this folder \install\langs\sr  language.xml is also one place with SR-CS that wrote in database SR-CS. I changed in file and in database SR-CS in SR-RS. And finely Serbian Latin language is working properly. It is possible now to use back office to translate it like other languages.
I am only worried now if someone do these changes in new prestashop verson that I have to translate it again from the very beginning. 
One more thing: I reported long ago that even file : \js\jquery\plugins\validate\localization\ messages_sr.js is wrong. Language used in this file is Serbian Cyrillic and it should be Latinic. This is not changed yet. I have provided correct file but nothing has been done. I can provide all files again if someone is willing to do these changes.

Kind regards
Zoran
